.github/workflows/test.yml

```diff
@@ -73,10 +73,14 @@ jobs:
         if: ${{ contains(matrix.os, 'windows') }}
         # MinGw64 and Git is already installed
         # https://github.com/actions/virtual-environments/blob/main/images/win/Windows2019-Readme.md
+        # Use gcc from cygwin for LP64 data model (not LLP64 with mingw)
         run: |
           choco install make -y
```

Comment:
Is this `make` still needed, since we now install the one from Cygwin below?

Reply:
good question, I'll try it

Reply:
Looks like it works, I'll merge it